go/host/in_mem_obscuro_client.go

```diff
@@ -213,12 +213,14 @@ func (c *inMemObscuroClient) Stop() {
 	// There is no RPC connection to close.
 }
 
-func (c *inMemObscuroClient) SetViewingKey(_ *ecies.PrivateKey, _ []byte) {
-	panic("viewing key encryption/decryption is not currently supported by in-memory obscuro-client")
+// viewing key encryption must be disabled on the enclave if using the in-memory client, it doesn't support viewing keys yet
+func (c *inMemObscuroClient) SetViewingKey(_ gethcommon.Address, _ *ecies.PrivateKey) {
+	// viewing key encryption/decryption is not currently supported by in-memory obscuro-client
```

Comment:
@BedrockSquirrel Here and below, either we should have a todo, or we should change the text to "is not supported by". Not clear to me whether we plan to add support or not, based on this.